plenum/test/node_catchup/test_node_request_consistency_proof.py

```diff
@@ -1,7 +1,4 @@
 import types
-from random import randint
-
-import pytest
 
 from plenum.common.constants import DOMAIN_LEDGER_ID, CONSISTENCY_PROOF
 from plenum.common.ledger import Ledger
```

Comment:
Could you also add simple test that compares root hashes of trees recovered using transaction log and using hash store?